force-app/main/default/lwc/nksVideoPlayer/nksVideoPlayer.html

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div if:true={isFileTypeMp4}>
```

Comment:
if:true brukes ikke lenger, bruk heller lwc:if https://developer.salesforce.com/docs/platform/lwc/guide/create-conditional.html

Reply:
Ser det er et par steder, men kommenterer bare det en gang. Det er ikke noe spes altså, if:true fungerer helt fint. Tenkte bare det var en nice to know ting